src/libstd/process.rs

```diff
@@ -605,6 +605,23 @@ pub struct ExitStatus(imp::ExitStatus);
 impl ExitStatus {
     /// Was termination successful? Signal termination not considered a success,
     /// and success is defined as a zero exit status.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use std::process::Command;
+    ///
+    /// let status = Command::new("/bin/mkdir")
```

Comment:
Could this perhaps be just "mkdir"? Surely we can assume that it is in the path, and it would avoid platform specific paths.